force-app/main/default/classes/AuroraSpikeController.cls

```diff
@@ -167,13 +167,49 @@ public with sharing class AuroraSpikeController {
         }
     }
     
+    public static string createAuroraProject(Site__c site){
+        System.debug('Site Id: ' + site.Id);
+        Aurora_Settings__c auroraSettings = Aurora_Settings__c.getInstance();
+        String tenantId  = auroraSettings.Tenant_Id__c;
+        // String tenantId  = '3d26171f-d1f3-4a9e-af93-9040f89e8fd1';
+        HttpRequest httpRequest = getAuroraEndpoint('/v2/tenants/' + tenantId + '/projects/' + designId + '/summary', 'POST'); 
```

Comment:
designId doesn't exist. Either need to have it passed in or the endpoint needs to be different.

Reply:
That doesn't need to be there. Removed it